chain/epoch_manager/src/lib.rs

```diff
@@ -173,7 +173,9 @@ impl EpochManager {
                     all_kicked_out = false;
                 }
             }
-            if block_stats.produced > maximum_block_prod && !is_already_kicked_out {
+            if (max_validator_id.is_none() || block_stats.produced > maximum_block_prod)
```

Comment:
is this equivalent to `block_stats.produced >= maximum_block_prod`?

Reply:
`>=` will be different when there's a tie

Reply:
this is equivalent to initializing `maximum_block_prod = -infinity`